conifer/converters/__init__.py

```diff
@@ -22,6 +24,74 @@ def get_converter(converter):
 def get_available_converters():
   return [k for k in _converter_map.keys()]
 
+
+def compute_float_lsb(config):
+    """Given a fixed point precision, compute the least significant bit.
+    If the precision is float, return None and let the other method compute
+    the smallest delta for each float.
+
+    Args:
+        config (Dict): backend configuration
+
+    Returns:
+        float|None: offset
+    """
+    if config is None or "Precision" not in config or config["Precision"] == "float":
+        # In floating point, the smallest difference depends on the magnitude of the number
+        # Use the nextafter numpy method
+        return None
+    ap_fixed = config["Precision"]
+    regex_match = re.search(r"ap_fixed<(\d+),(\d+)(,|>)", ap_fixed)
```

Comment:
Here I suggest to use the following approach to get the LSB value:
```
import conifer
fpc = conifer.utils.FixedPointConverter(precision)
lsb_value = fpc.from_int(1)
```
This would automatically support the different variations like `ap_ufixed`, `ap_int`, `ap_uint` for example.